scripts/import_dataset.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -e
+# Copyright 2020. IBM All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BASEDIR="$(dirname "$0")"
+source ${BASEDIR}/env.sh
+
+# Install aria and get example dataset zip file
+echo "Installing aria2..."
+apt-get -o Dpkg::Use-Pty=0 update -qq  || echo " RC${?} Got an error on update???"
+apt-get -o Dpkg::Use-Pty=0 install -qq aria2
+
+cd $HOME
```

Comment:
Hey! don't leave stuff in $HOME; instead use /tmp and tidy up after yourself at the end of the script by deleting the file.

Reply:
Oh okay, I wasn't sure if it would be better to leave the file for them there in case they wanted to re-upload it again at some point to play around with the dataset and then maybe expand the comment to include that information? Not sure what the typical use case would be/how long they would have the instance up.